pkg/login/social/grafana_com_oauth.go

```diff
@@ -49,6 +49,10 @@ func (s *SocialGrafanaCom) IsOrganizationMember(organizations []OrgRecord) bool
 	return false
 }
 
+func (s *SocialGrafanaCom) OrgToRoleMap() map[string][]SocialGroup {
+	return nil
+}
+
```

Comment:
I think we should change other oauth providers if not needed in order to keep the patch as small as possible and make it easier to merge upstream changes